### DISCUSSION TITLE
my-usage-reports-of-unraid

### DISCUSSION THREAD

**Discussion by bowencool** [bowencool/blog#36] [Announcements]
# my-usage-reports-of-unraid

我的 unRAID 使用报告（2022 至 2023 ）

https://blog.bowen.cool/zh/posts/my-usage-reports-of-unraid

**Comment by ChasLui** [comment]
我使用 zerotier + 国内服务器部署moon, 使用起来很方便. 基本ping在10-20ms.

**Comment by bowencool** [comment]
> 我使用 zerotier + 国内服务器部署moon, 使用起来很方便. 基本ping在10-20ms.

zerotier 好像很有用，但我添加的静态路由（家里内网）不知道咋不生效。Moon 是什么，有链接吗

**Comment by ChasLui** [comment]
> > 我使用 zerotier + 国内服务器部署moon, 使用起来很方便. 基本ping在10-20ms.
> 
> zerotier 好像很有用，但我添加的静态路由（家里内网）不知道咋不生效。Moon 是什么，有链接吗

[Zerotier Moon服务器搭建](https://blog.timzhong.top/2021/02/01/zerotier-moon-server/)
我使用的是https://supes.top/编译的openwrt纯净版系统内置了zerotier客户端, 

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/10083758/221763411-cbed829b-7e96-4c74-957a-64eaf6eabad0.png">

然后用ssh连上openwrt,执行`zerotier-cli orbit 你的moonid 你的moonid`, 每个zerotier客户端都得用orbit加入,所有像ios就没法用moon中转服务.

**Comment by bowencool** [comment]
> > > 我使用 zerotier + 国内服务器部署moon, 使用起来很方便. 基本ping在10-20ms.
> > 
> > 
> > zerotier 好像很有用，但我添加的静态路由（家里内网）不知道咋不生效。Moon 是什么，有链接吗
> 
> [Zerotier Moon服务器搭建](https://blog.timzhong.top/2021/02/01/zerotier-moon-server/) 我使用的是[https://supes.top/编译的openwrt纯净版系统内置了zerotier客户端](https://supes.top/%E7%BC%96%E8%AF%91%E7%9A%84openwrt%E7%BA%AF%E5%87%80%E7%89%88%E7%B3%BB%E7%BB%9F%E5%86%85%E7%BD%AE%E4%BA%86zerotier%E5%AE%A2%E6%88%B7%E7%AB%AF),
> 
> <img alt="image" width="1062" src="https://user-images.githubusercontent.com/10083758/221763411-cbed829b-7e96-4c74-957a-64eaf6eabad0.png">
> 
> 然后用ssh连上openwrt,执行`zerotier-cli orbit 你的moonid 你的moonid`, 每个zerotier客户端都得用orbit加入,所有像ios就没法用moon中转服务.

我也是这样配的静态路由，但就是访问不到，搞不懂。我还是用 IPv6 直接连吧

**Comment by Kyvrel** [comment]
小米云有同步的接口吗

**Comment by bowencool** [comment]
没有，和 iCloud一样都是从网页版扒的接口，你可以试试这个 [xiaomi cloud](https://github.com/sunhanaix/SyncXiaomiCloud)

**Comment by sethome2** [comment]
第一次看到有人在issue里面写博客，震惊，我还以为的打开到了blog的评论区，找来找去也没看到page在哪，这才反应过来。在找Tailscale访问不了unraid webui的问题。补充说一下，unraid好像没有官方的包管理器了（目前6.12.3），社区有插件可以装apt和yum，有点诡异。

**Comment by ZvonimirSun** [comment]
> 第一次看到有人在issue里面写博客，震惊，我还以为的打开到了blog的评论区，找来找去也没看到page在哪，这才反应过来。在找Tailscale访问不了unraid webui的问题。补充说一下，unraid好像没有官方的包管理器了（目前6.12.3），社区有插件可以装apt和yum，有点诡异。

啊，之前是有包管理器的吗，6.11的时候我看就没有，不过也还好，主要用docker

**Comment by ZvonimirSun** [comment]
nextcloud之前装过，虽然功能确实多，不过最基础的文件同步经常有问题，用不起来。

推荐seafile，专业版免费3用户，多大的文件同步都没出现过问题，也能搭配onlyoffice，作为网盘的基础功能非常稳定。就是界面感觉没有那么现代，功能没有那么丰富，不过胜在简洁，基础牢靠

**Comment by bowencool** [comment]
> 第一次看到有人在issue里面写博客，震惊，我还以为的打开到了blog的评论区，找来找去也没看到page在哪，这才反应过来。在找Tailscale访问不了unraid webui的问题。补充说一下，unraid好像没有官方的包管理器了（目前6.12.3），社区有插件可以装apt和yum，有点诡异。

我都是用 NerdTools 装其他包的，也就 vim git 等

**Comment by bowencool** [comment]
> 



> nextcloud之前装过，虽然功能确实多，不过最基础的文件同步经常有问题，用不起来。
> 
> 推荐seafile，专业版免费3用户，多大的文件同步都没出现过问题，也能搭配onlyoffice，作为网盘的基础功能非常稳定。就是界面感觉没有那么现代，功能没有那么丰富，不过胜在简洁，基础牢靠

我已经不相信这些网盘应用了，现在依靠 rclone + webdav 做文件同步，主要是自动化同步软件/应用/扩展程序的配置文件，普通文件我直接OpenVPN/Tailscale回内网取。

**Comment by ZvonimirSun** [comment]
> > 
> 
> > nextcloud之前装过，虽然功能确实多，不过最基础的文件同步经常有问题，用不起来。
> > 推荐seafile，专业版免费3用户，多大的文件同步都没出现过问题，也能搭配onlyoffice，作为网盘的基础功能非常稳定。就是界面感觉没有那么现代，功能没有那么丰富，不过胜在简洁，基础牢靠
> 
> 我已经不相信这些网盘应用了，现在依靠 rclone + webdav 做文件同步，主要是自动化同步软件/应用/扩展程序的配置文件，普通文件我直接OpenVPN/Tailscale回内网取。

seafile专门搞企业云盘的，和nextcloud这种社区导向的不好比，很多企业在用，而且c的应用和php的应用稳定性不在一个层次，我同步了几百个G暂时也没见过失败的。像挂载盘、文件历史版本之类的功能，还得靠这种企业级的来做。

哦对，rclone也支持直接接seafile，感觉比webdav方案稳定。

**Comment by irvinow** [comment]
您好，能请教一下rclone的问题么，我已经用rclone连接上百度云盘了，但是写的script不能备份，请问script口令怎么写？

**Comment by bowencool** [comment]
> 您好，能请教一下rclone的问题么，我已经用rclone连接上百度云盘了，但是写的script不能备份，请问script口令怎么写？

刚才发现链接有问题，你看看这个呢：https://github.com/bowencool/blog/issues/18

**Comment by irvinow** [comment]
> > 您好，能请教一下rclone的问题么，我已经用rclone连接上百度云盘了，但是写的script不能备份，请问script口令怎么写？
> 
> 
> 
> 刚才发现链接有问题，你看看这个呢：https://github.com/bowencool/blog/issues/18

谢谢，我也在抄作业学习状态

**Comment by irvinow** [comment]
deepl free api还能用吗？谢谢

**Comment by bowencool** [comment]
> deepl free api还能用吗？谢谢

能用

**Comment by bowencool** [comment]
> > 我已经不相信这些网盘应用了，现在依靠 rclone + webdav 做文件同步，主要是自动化同步软件/应用/扩展程序的配置文件，普通文件我直接OpenVPN/Tailscale回内网取。
> 
> seafile专门搞企业云盘的，和nextcloud这种社区导向的不好比，很多企业在用，而且c的应用和php的应用稳定性不在一个层次，我同步了几百个G暂时也没见过失败的。像挂载盘、文件历史版本之类的功能，还得靠这种企业级的来做。
> 
> 哦对，rclone也支持直接接seafile，感觉比webdav方案稳定。

我试了一下 Seafile，发现它自己实现了一套 fs，有点像 Git。

<img width="764" alt="image" src="https://github.com/bowencool/blog/assets/20217146/2d946675-f7ac-4e4b-8ffc-b1ab639921d2">

是不是说，我在 NAS 上安装了服务端，但我也没法直接访问文件，必须在 NAS 上再装一个客户端？
